### PR TITLE
docs: update the PyPI URL & minor wording

### PR DESCRIPTION
**Pull request by toolifelesstocode**
## Summary

This pull request just updates the README.md file a bit.

- Added hyphens in words where it's needed
- Updated the PyPI URL, as the domain had changed a few months ago

## Checklist

- [ ] If code changes were made, then they have been tested
    - [ ] I have updated the documentation to reflect the changes
    - [ ] I have formatted the code properly by running `task lint`
    - [ ] I have type-checked the code by running `task pyright`
- [ ] This PR fixes an issue
- [ ] This PR adds something new (e.g. new method or parameters)
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [x] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,12 +5,12 @@ disnake
...
-    <a href="https://pypi.python.org/pypi/disnake"><img src="https://img.shields.io/pypi/v/disnake.svg?style=flat-square" alt="PyPI version info" /></a>
-    <a href="https://pypi.python.org/pypi/disnake"><img src="https://img.shields.io/pypi/pyversions/disnake.svg?style=flat-square" alt="PyPI supported Python versions" /></a>
+    <a href="https://pypi.org/project/disnake/"><img src="https://img.shields.io/pypi/v/disnake.svg?style=flat-square" alt="PyPI version info" /></a>
+    <a href="https://pypi.org/project/disnake/"><img src="https://img.shields.io/pypi/pyversions/disnake.svg?style=flat-square" alt="PyPI supported Python versions" /></a>
     <a href="https://github.com/DisnakeDev/disnake/commits"><img src="https://img.shields.io/github/commit-activity/w/DisnakeDev/disnake.svg?style=flat-square" alt="Commit activity" /></a>
 </p>
 
-A modern, easy to use, feature-rich, and async ready API wrapper for Discord written in Python.
+A modern, easy-to-use, feature-rich, and async-ready API wrapper for Discord written in Python.
```
Comment (shiftinv) [inline README.md:13]: should be changed in `docs/index.rst` as well, for consistency

Comment (toolifelesstocode) [reply README.md:13]: 👍🏼

**Comment by ItsAleph** [comment]
> ... a few months ago ...

?
AFAIK this URL was used everywhere all the time I were in Python (at least last 3-4 years). :l
Just as a general question, nothing special

**Comment by onerandomusername** [comment]
> > ... a few months ago ...
> 
> ? AFAIK this URL was used everywhere all the time I were in Python (at least last 3-4 years). :l Just as a general question, nothing special

It was actually [2018](https://github.com/pypa/pypi-legacy#no-longer-in-service)

**Comment by toolifelesstocode** [comment]
Okay, that's fine, it doesn't really matter much when it changed, but yes the URL did change at some point.

File: docs/index.rst [hunk 2]
```diff
@@ -9,15 +9,15 @@ Welcome to disnake
 .. image:: /images/snake.svg
 .. image:: /images/snake_dark.svg
 
-disnake is a modern, easy to use, feature-rich, and async ready API wrapper
+disnake is a modern, easy-to-use, feature-rich, and async-ready API wrapper
```
Comment (onerandomusername) [inline docs/index.rst:12]: Not entirely sold on the easy-to-use changes here.

File: docs/index.rst [hunk 3]
```diff
@@ -9,15 +9,15 @@ Welcome to disnake
...
 for Discord.
 
 **Features:**
 
 - Modern Pythonic API using ``async``\/``await`` syntax
-- Sane rate limit handling that prevents 429s
+- Sane rate limit handling that prevents 429 errors
 - Command extension to aid with bot creation
-- Easy to use with an object oriented design
+- Easy-to-use with an object oriented design
```
Comment (onerandomusername) [inline docs/index.rst:20]: same here

File: README.md (hunk 1 above)
Comment (onerandomusername) [inline README.md:13]: same here

**Review by onerandomusername — CHANGES_REQUESTED** [review]
@shiftinv what are your thoughts?

File: docs/index.rst (hunk 2 above)
Comment (shiftinv) [reply docs/index.rst:12]: +1, "easy to use" seems better here

Comment (onerandomusername) [reply docs/index.rst:12]: @toolifelesstocode would you please update this?

Comment (toolifelesstocode) [reply docs/index.rst]: All fixed.

**Review by onerandomusername — APPROVED** [review]
thanks for catching these

**Pull request merged**